openstack/ot/jaeger.go

```diff
@@ -0,0 +1,43 @@
+package ot
```

Comment:
Please delete this file. As mentioned in a previous comment (see "No opinionated tracing in Gophercloud"), tracing can be added right from the application; I see no need for it in the library.